martin-mbtiles/src/tile_copier.rs

```diff
@@ -98,23 +98,40 @@ impl TileCopier {
             .execute(&mut conn)
             .await?;
 
-        let schema = query("SELECT sql FROM sourceDb.sqlite_schema WHERE tbl_name IN ('metadata', 'tiles', 'map', 'images')")
-            .fetch_all(&mut conn)
-            .await?;
-
-        for row in &schema {
-            let row: String = row.get(0);
-            query(row.as_str()).execute(&mut conn).await?;
-        }
+        if !self.options.force_simple {
+            for row in query("SELECT sql FROM sourceDb.sqlite_schema WHERE tbl_name IN ('metadata', 'tiles', 'map', 'images')")
```

Comment:
would this include copying indexes? If not, you may need a second statement like this to copy index creation

Reply:
it does, because I don't specify the type of the schema statement, it will take statements where `type='index'`